lib/services/message-relay.js

```diff
@@ -5,6 +5,7 @@ class MessageRelay extends EventEmitter {
   constructor() {
     super();
     this.listeners = {};
+    this.pollActive = false;
```

Comment:
I think tracking poll state here makes writing tests for the gulag and voteban commands more difficult, but it doesn't quite fit here regardless. It makes more sense to relay `POLLSTART` messages to listeners as well and have them track this independently.